packages/storage/amplify_storage_s3_dart/lib/src/model/s3_download_data_options.dart

```diff
@@ -53,4 +58,10 @@ class S3DownloadDataOptions extends StorageDownloadDataOptions {
   /// Whether to retrieve properties for the downloaded object using the
   /// `getProperties` API.
   final bool getProperties;
+
+  /// {@template storage.amplify_storage_s3.transfer_acceleration}
```

Comment:
nit/question: Nice comment, would it be possible to have same comment for download_file, upload_file/data, etc...?

Reply:
This is a good thought.

Developers are interacting with the Storage APIs via the Storage Category API interface, the documentation should be added here. The category interface, however, hides the S3 specifics, i.e. transfer acceleration is S3 unique. So I think the discoverability of this configuration would depend on Amplify library documentation and Flutter/Dart API docs on pub.dev. What do you think?